python/arcticdb/version_store/_normalization.py

```diff
@@ -516,7 +516,7 @@ def _normalize_columns_names(columns_names, index_names, norm_meta, dynamic_sche
             if dynamic_schema and (counter[col] > 1):
                 raise ArcticNativeException("Same column names not allowed in dynamic_schema")
             new_name = col_str
-            if counter[col] > 1 or col in index_names:
```

Comment:
I'm now not sure that this is a great idea. I should double check what happens when we have duplicated column names that also match the index names